### DISCUSSION TITLE
viewservlets 4.9.0 on Maven?

### DISCUSSION THREAD

**Discussion by JacquesLeRoux** [eclipse-birt/birt#863] [Q&A] [answered]
Hi,

In Apache OFBiz we are using the Birt runtime viewservlets since 2009. [Since 2016](https://github.com/apache/ofbiz-plugins/blob/trunk/birt/build.gradle), we are stuck with [4.5.0 ](https://mvnrepository.com/artifact/org.eclipse.birt.runtime/viewservlets). We are currently facing a security report related to this bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=538142 fixed with https://github.com/eclipse/birt/pull/473/commits/f119e6e680aa9af123b9ec732154e67b70144ebe.

I know the 4.9.0 version of Birt will be released tomorrow. Is there a hope that the viewservlets 4.9.0 will be released on Maven soon?

TIA

**Comment by wimjongman** [comment, marked as answer]
We have issue #625 open for this. Hopefully, we can revive this and add the artifacts as soon as possible.

**Comment by sids0501** [comment]
Anyone can help with information. I don't see viewservlet 4.9.0. What was the outcome to resolve vulneribilities on 4.5.0 dependency

**Comment by JacquesLeRoux** [comment]
I tried,  could not find a solution, see https://github.com/eureka-bpo/p2repository.com/issues/2#issuecomment-2568988080

**Comment by JacquesLeRoux** [comment]
To be more clear: [Merks's answer clarifies things](https://github.com/eclipse-birt/birt/issues/625#issuecomment-2772505317)
I have not tested that yet...
